frontend/nuxt.config.ts

```diff
@@ -46,7 +46,8 @@ export default defineNuxtConfig({
     },
     private: {
       apiServer: process.env.PRIVATE_API_SERVER,
-      legacyApiServer: process.env.PRIVATE_LEGACY_API_SERVER
+      legacyApiServer: process.env.PRIVATE_LEGACY_API_SERVER,
+      ssrSecret: process.env.PRIVATE_SSR_SECRET || ''
```

Comment:
nitpick:
```suggestion
      ssrSecret: ''
```
Except `ssrSecret` should be `undefined` on purpose